app/js/components/manageApps/Addon.jsx

```diff
@@ -115,29 +114,61 @@ class Addon extends Component {
     });
   }
 
-  handleModuleDelete() {
+  handleModuleDelete(event, moduleUuid, action) {
```

Comment:
`moduleUuid` and `action` are passed in here but are not used

Reply:
Also `handleModuleDelete` and `handleOwaDelete` are basically the same. Have one `handleAddonDelete` to this functionality

Reply:
Removed the unnecessary function parameters